packages/react-core/src/Button/index.js

```diff
@@ -94,7 +95,13 @@ const Button = styled(
       specificProps = { disabled, type };
     }
 
-    return <Tag {...specificProps} {...props} />;
+    return (
+      <Tag {...specificProps} {...props}>
+        {React.Children.map(children, node =>
+          typeof node === 'string' ? <span>{node}</span> : node
```

Comment:
Unfortunately there's no way to use `:first-child` and `:last-child` if the text nodes are not wrapped by an HTML tag.

Reply:
I guess you can use span for number as well?